test/e2e/servicebindingrequest_test.go

```diff
@@ -63,26 +65,17 @@ func TestAddSchemesToFramework(t *testing.T) {
 		t.Run("scenario-db-app-sbr", func(t *testing.T) {
 			ServiceBindingRequest(t, []Step{DBStep, AppStep, SBRStep})
 		})
-
```

Comment:
@isutton Why have you removed those scenarios? Those implements the e2e step permutations as described in https://jira.coreos.com/browse/APPSVC-126.

Is it intentional or accidental?